src/components/EditCoursePage/index.jsx

```diff
@@ -126,19 +125,31 @@ class EditCoursePage extends React.Component {
     const {
       editCourse,
     } = this.props;
-    // Create a deep copy of course runs since we modify their properties
-    let modifiedCourseRuns = jsonDeepCopy(courseData.course_runs);
 
-    // Don't send any courses in review - backend will reject them
-    modifiedCourseRuns = modifiedCourseRuns.filter(run => !IN_REVIEW_STATUS.includes(run.status));
+    const { targetRun } = this.state;
 
-    // Process course run info
-    this.prepareCourseRuns(modifiedCourseRuns, courseData);
+    // Process course run info from courseData
+    const modifiedCourseRuns = this.prepareCourseRuns(courseData);
+    // Update the Draft status based on run status
+    let courseEditData = jsonDeepCopy(courseData); // eslint-disable-line prefer-const
+    if (targetRun) {
```

Comment:
Only deepcopying once will also allow this logic to move back into the `prepareCourseRuns` method, which I think I prefer because it's purpose is deciding which courseRun is the targetRun and that should fall under that helper function.